### PR TITLE
make IQuesionnaireSchema match questionnaire.schema.json

### PR DESCRIPTION
**Pull request by lucas-sdl**
-- haven't pushed packages yet

### CODE REVIEW

File: documentation/questionnaires/questionnaire.schema.json [hunk 1]
```diff
@@ -41,15 +41,15 @@
...
-                        "description": "The label for the lower end of the lineer scale"
+                        "description": "The label for the lower end of the linear scale"
                     },
                     "upperLabel": {
                         "type": "string",
-                        "description": "The label for the upper end of the lineer scale"
+                        "description": "The label for the upper end of the linear scale"
                     },
-                    "minumum": {
+                    "minimum": {
```
Comment (SnickerChar) [inline documentation/questionnaires/questionnaire.schema.json:50]: Good catch on the typo. I totally missed that

File: packages/extension-onboarding/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@snickerdoodlelabs/extension-onboarding",
-  "version": "1.0.0",
+  "version": "1.0.1",
```
Comment (SnickerChar) [inline packages/extension-onboarding/package.json:3]: This version isn't used for anything, FYI

**Review by SnickerChar — APPROVED** [review]
Works for me.

**Review by moahammadalt — APPROVED** [review]

File: packages/extension-onboarding/package.json (hunk 2 above)
Comment (lucas-sdl) [reply packages/extension-onboarding/package.json:3]: Good to know

File: documentation/questionnaires/questionnaire.schema.json (hunk 1 above)
Comment (lucas-sdl) [reply documentation/questionnaires/questionnaire.schema.json:50]: All thanks goes to my ide spell checker

**Review by orhunerd — APPROVED** [review]

**Comment by lucas-sdl** [comment]
this was taken care of in another PR

**Pull request closed without merging**